.env.example

```diff
@@ -6,4 +6,9 @@ ACCESS_TOKEN_COOKIE_SECURE=
 ACCESS_TOKEN_COOKIE_SAME_SITE=
 ACCESS_TOKEN_COOKIE_DOMAIN=
 ACCESS_TOKEN_COOKIE_PATH=
-ACCESS_TOKEN_COOKIE_MAX_AGE=
\ No newline at end of file
+ACCESS_TOKEN_COOKIE_MAX_AGE=
+
+# PostgreSQL Database Configuration
+POSTGRESQL_DB_URL=
+POSTGRESQL_DB_USERNAME=
+POSTGRESQL_DB_PASSWORD=
```

Comment:
Check de nieuwste code van Daan, heeft nu namelijk andere env variables. Die heeft er nu namelijk 6 ofzo en die structuur moet je overnemen als je wilt dat je db werkt in openshift. Kijk ook naar zijn application.properties